pybedtools/bedtool.py

```diff
@@ -1403,10 +1403,17 @@ def _collapse(
         # the files directly
         if isinstance(iterable, BedTool) and isinstance(iterable.fn, six.string_types):
             with out_open_func(fn, "wt") as out_:
-                with in_open_func(iterable.fn, "rt") as in_:
-                    if trackline:
-                        out_.write(trackline.strip() + "\n")
+                in_ = in_open_func(iterable.fn, "rt")
+                if trackline:
+                    out_.write(trackline.strip() + "\n")
+                try:
                     out_.writelines(in_)
+                except UnicodeDecodeError:
+                    in_.close()
+                    with in_open_func(iterable.fn, "rt", errors="ignore") as in_:
+                        out_.writelines(in_)
```

Comment:
If the error isn't seen until e.g. line 100 out of 1000, won't this cause the output file have 100 + 1000 lines?